packages/manager/apps/cloud/client/app/app.js

```diff
@@ -172,6 +172,9 @@ angular.module('managerApp', [
   .config(/* @ngInject */ (CucConfigProvider, coreConfigProvider) => {
     CucConfigProvider.setRegion(coreConfigProvider.getRegion());
   })
+  .config(/* @ngInject */ (ouiCalendarConfigurationProvider, TranslateServiceProvider) => {
+    ouiCalendarConfigurationProvider.setLocale(TranslateServiceProvider.getUserLocale(true));
```

Comment:
Does it make sense to add this configuration into `@ovh-ux/manager-core` instead ?

Reply:
Thats a good idea :)
This is done 👍